packages/ios-client/Sources/FishjamClient/FishjamClientInternal.swift

```diff
@@ -750,7 +750,7 @@ extension FishjamClientInternal: RTCEngineListener {
     }
 
     func onVadNotification(trackId: String, status: String) {
-        guard let track = getTrack(trackId: trackId) else {
+        guard let track = getTrackWithRtcEngineId(trackId: trackId) else {
```

Comment:
Maybe we should put `as? RemoteAudioTrack` here instead of casting it twice later on. Seems like this is how it works on Android as well